<launch>
    <!-- Start pan joint controller -->
    <rosparam file="$(find gripper)/pan.yaml" command="load"/>
    <node name="pan_controller_spawner" pkg="dynamixel_controllers" type="controller_spawner.py"
          args="--manager=dxl_manager
                --port your_dynamixel_usb_port
                pan_controller"
          output="screen"/>
</launch>
